Hunting Queries/SecurityEvent/LargeScaleMalwareDeploymentGPOScheduledTask.yaml

```diff
@@ -0,0 +1,27 @@
+id: a1a06ba2-87f8-11ec-a8a3-0242ac120002
+name: Large Scale Malware Deployment via GPO Scheduled Task Modification
+description: |
+  'This query detects lateral movement using GPO scheduled task usually used to deploy ransomware at scale.
+   It monitors whether a scheduled task is modified within the Sysvol folder in GPO.
+   Ref: https://bogusecurity.com/2019/12/26/persistence-and-execution-at-scale-via-gpo-scheduled-task/'
+severity: High
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - LateralMovement
+relevantTechniques:
+  - T1484
+query: |
+ SecurityEvent
+    | where EventID == 5145 
+    | parse EventData with * 'ShareName">' ShareName "<" *
+    | parse EventData with * 'RelativeTargetName">' RelativeTargetName "<" *
+    | parse EventData with * 'AccessList">' AccessList "<" *
+    | parse EventData with * 'IpAddress">' IpAddress "<" *
+    | parse EventData with * 'SubjectUserName">' SubjectUserName "<" *
+    | parse EventData with * 'SubjectDomainName">' SubjectDomainName "<" *
+    | parse EventData with * 'SubjectLogonId">' SubjectLogonId "<" *
+    | where ShareName == "\\\\*\\SYSVOL" and RelativeTargetName endswith "ScheduledTasks.xml" and AccessList contains "%%4417"
```

Comment:
It seems that many of the fields like ShareName, RelativeTargetName etc. are natively available. Not sure if there is a need to parse them. Can you please double check if these parse statements are needed?

Reply:
Thanks for the changes